pyrometheus/__init__.py

```diff
@@ -63,52 +67,84 @@ def str_np_inner(ary):
 def str_np(ary):
     return "np.array(%s)" % str_np_inner(ary)
 
+
 # }}}
 
 
 # {{{ polynomial processing
 
+
 def nasa7_conditional(t, poly, part_gen):
     # FIXME: Should check minTemp, maxTemp
     return p.If(
-            p.Comparison(t, "<", poly.coeffs[0]),
-            part_gen(poly.coeffs[1:8], t),
-            part_gen(poly.coeffs[8:15], t))
+        p.Comparison(t, ">", poly.coeffs[0]),
+        part_gen(poly.coeffs[1:8], t),
+        part_gen(poly.coeffs[8:15], t),
+    )
 
 
 @singledispatch
 def poly_to_expr(poly):
-    raise TypeError(f"unexpected argument type in poly_to_expr: {type(poly)}")
+    message = f"unexpected argument type in poly_to_expr: {type(poly)}"
+    raise TypeError(message)
 
 
 @poly_to_expr.register
 def _(poly: ct.NasaPoly2, arg_name):
     def gen(c, t):
         assert len(c) == 7
-        return c[0] + c[1]*t + c[2]*t**2 + c[3]*t**3 + c[4]*t**4
+        return c[0] + c[1] * t + c[2] * t ** 2 + c[3] * t ** 3 + c[4] * t ** 4
 
     return nasa7_conditional(p.Variable(arg_name), poly, gen)
 
 
 @singledispatch
 def poly_to_integral_expr(poly, arg_name):
-    raise TypeError(
-            f"unexpected argument type in poly_to_integral_expr: {type(poly)}")
+    message = f"unexpected argument type in poly_to_integral_expr: {type(poly)}"
+    raise TypeError(message)
 
 
 @poly_to_integral_expr.register
 def _(poly: ct.NasaPoly2, arg_name):
     def gen(c, t):
         assert len(c) == 7
-        return c[0]*t + c[1]/2*t**2 + c[2]/3*t**3 + c[3]/4*t**4 + c[4]/5*t**5
+        return (
+            c[0] * t
+            + c[1] / 2 * t ** 2
+            + c[2] / 3 * t ** 3
+            + c[3] / 4 * t ** 4
+            + c[4] / 5 * t ** 5
+        )
+
+    return nasa7_conditional(p.Variable(arg_name), poly, gen)
```

Comment:
Since you added `poly_to_enthalpy_expr`, it seems that this is now unused. Why not delete it?

Reply:
Ok, will be gone by next commit.